negotiation/message/shape/contract-agreement-verification-message-shape.ttl

```diff
@@ -25,28 +25,18 @@ shapes:
 	] ;
 .
 
-shapes:ContractAgreementVerificationMessageShape
+dspace_shapes:ContractAgreementVerificationMessageShape
 	a sh:NodeShape ;
 	sh:targetClass dspace:ContractAgreementVerificationMessage ;
-	rdfs:subClassOf shapes:ContractNegotiationMessageShape ;
+	rdfs:subClassOf dspace_shapes:ContractNegotiationMessageShape ;
 
 	sh:property [
 		a sh:PropertyShape ;
-		sh:path cred:credentialSubject ;
-		sh:class dspace:ContractAgreementMessage ; # TODO: Having the ContractAgreementMessage class(!) here but only with the property dspace:hash is a bad idea, not safe for the schema engine.
+        sh:path dspace:processPid ;
```

Comment:
Please see Julia's comment: https://github.com/International-Data-Spaces-Association/ids-specification/pull/89#discussion_r1339693532 
`dspace:processPid` needs to be replaced with `dspace:providerPid` and `dspace:consumerPid`.